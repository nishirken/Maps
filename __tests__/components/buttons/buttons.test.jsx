import { Buttons } from 'Components';

describe('Buttons in the marks list item', () => {
    const TestButtons = shallow(<Buttons />);

    it('Rendered', () => {
        expect(TestButtons).toMatchSnapshot();
    });

    it('Has an edit button', () => {
        expect(TestButtons.find('EditButton').length).toBe(1);
    });

    it('Has a delete button', () => {
        expect(TestButtons.find('DeleteButton').length).toBe(1);
    });
});
