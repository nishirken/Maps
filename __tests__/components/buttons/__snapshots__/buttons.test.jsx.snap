// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons in the marks list item Rendered 1`] = `
<styled.div>
  <EditButton />
  <DeleteButton />
</styled.div>
`;
